From 90d074f764d249cda61af737e194c20323f367fa Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Mon, 4 Jun 2018 14:55:49 +0800
Subject: [PATCH] arm: dts: rockchip: thermal: update soc's sw/hw over
 temperature power off degree

to cope with Wide Temperature Range test, we maxamize
soc's sw/hw over temperature power off degree.

fow now, 115 degree Celsius is set to trigger sw powering off.
if sw function does not work and temperature is continuing to
grow up, and till 120 degree Celsius, hw powering off/reset
is triggered.

Change-Id: I751e9ea754f434bc20df39fdbdb40216a1582c39
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
---
 arch/arm/boot/dts/rk3128h-box.dts                        | 2 +-
 arch/arm/boot/dts/rk3128x.dtsi                           | 4 ++--
 arch/arm/boot/dts/rk312x.dtsi                            | 2 +-
 arch/arm/boot/dts/rk3229-evb-android.dts                 | 2 +-
 arch/arm/boot/dts/rk322x.dtsi                            | 4 ++--
 arch/arm/boot/dts/rk3288.dtsi                            | 4 ++--
 arch/arm64/boot/dts/rockchip/px30.dtsi                   | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3308.dtsi                 | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dts     | 2 +-
 arch/arm64/boot/dts/rockchip/rk3328-evb-android.dts      | 2 +-
 arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dts   | 2 +-
 arch/arm64/boot/dts/rockchip/rk3328.dtsi                 | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3366.dtsi                 | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3368.dtsi                 | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi             | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts    | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts      | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts       | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399.dtsi                 | 4 ++--
 20 files changed, 33 insertions(+), 33 deletions(-)

diff --git a/arch/arm/boot/dts/rk3128h-box.dts b/arch/arm/boot/dts/rk3128h-box.dts
index f69a4744b318..a8816cbae598 100644
--- a/arch/arm/boot/dts/rk3128h-box.dts
+++ b/arch/arm/boot/dts/rk3128h-box.dts
@@ -566,7 +566,7 @@
 };
 
 &soc_crit {
-	temperature = <110000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
diff --git a/arch/arm/boot/dts/rk3128x.dtsi b/arch/arm/boot/dts/rk3128x.dtsi
index 3514e23d93f4..bed1a10079bb 100644
--- a/arch/arm/boot/dts/rk3128x.dtsi
+++ b/arch/arm/boot/dts/rk3128x.dtsi
@@ -664,7 +664,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <90000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -714,7 +714,7 @@
 		pinctrl-1 = <&otp_out>;
 		pinctrl-2 = <&otp_gpio>;
 		#thermal-sensor-cells = <0>;
-		rockchip,hw-tshut-temp = <95000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 677b535c1c4c..c4e7bd2cb45a 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -291,7 +291,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <100000>;
+					temperature = <115000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
diff --git a/arch/arm/boot/dts/rk3229-evb-android.dts b/arch/arm/boot/dts/rk3229-evb-android.dts
index 6ab0a5ea4cf4..7b2d62063622 100644
--- a/arch/arm/boot/dts/rk3229-evb-android.dts
+++ b/arch/arm/boot/dts/rk3229-evb-android.dts
@@ -491,7 +491,7 @@
 };
 
 &soc_crit {
-	temperature = <110000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index c88af702e904..4baa078933da 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -681,7 +681,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <90000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -731,7 +731,7 @@
 		pinctrl-1 = <&otp_out>;
 		pinctrl-2 = <&otp_gpio>;
 		#thermal-sensor-cells = <0>;
-		rockchip,hw-tshut-temp = <95000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index 4a428bfed5e1..e203912d60a9 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -602,7 +602,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <90000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -646,7 +646,7 @@
 		pinctrl-1 = <&otp_gpio>;
 		pinctrl-2 = <&otp_gpio>;
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <95000>;
+		rockchip,hw-tshut-temp = <120000>;
 		rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
 		status = "disabled";
 	};
diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index d4254f0b4359..39cc9cbf8849 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -905,7 +905,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>;
+					temperature = <115000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -945,7 +945,7 @@
 		resets = <&cru SRST_TSADC>;
 		reset-names = "tsadc-apb";
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <100000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index 4b9111855789..f6289169728a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -632,7 +632,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>;
+					temperature = <115000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
@@ -672,7 +672,7 @@
 		pinctrl-1 = <&tsadc_otp_out>;
 		pinctrl-2 = <&tsadc_otp_gpio>;
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <100000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dts b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dts
index 3f96e6436b27..607a03eb6de6 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dts
@@ -536,7 +536,7 @@
 };
 
 &soc_crit {
-	temperature = <110000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dts b/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dts
index 44b37ce5f4cb..9541778bc17b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dts
@@ -611,7 +611,7 @@
 };
 
 &soc_crit {
-	temperature = <110000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dts b/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dts
index 8ae0074310c7..fa626fb24e37 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dts
@@ -520,7 +520,7 @@
 };
 
 &soc_crit {
-	temperature = <110000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index 6ed8f40b1116..7c7e715b7c34 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -382,7 +382,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -430,7 +430,7 @@
 		pinctrl-1 = <&otp_out>;
 		pinctrl-2 = <&otp_gpio>;
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <100000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3366.dtsi b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
index 927daa725235..91a9adccb88e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3366.dtsi
@@ -298,7 +298,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -337,7 +337,7 @@
 		pinctrl-names = "default";
 		pinctrl-0 = <&tsadc_gpio>;
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <95000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index c5277499125e..46617af93339 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -725,7 +725,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
index 66bf58dc9616..87b303503e35 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
@@ -541,7 +541,7 @@
 };
 
 &soc_crit {
-	temperature = <105000>;
+	temperature = <115000>;
 };
 
 &tcphy0 {
@@ -558,7 +558,7 @@
 	rockchip,hw-tshut-mode = <1>;
 	/* tshut polarity 0:LOW 1:HIGH */
 	rockchip,hw-tshut-polarity = <1>;
-	rockchip,hw-tshut-temp = <110000>;
+	rockchip,hw-tshut-temp = <120000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index a146ff5cb0fc..8dc6e78eac64 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -525,7 +525,7 @@
 };
 
 &soc_crit {
-	temperature = <105000>;
+	temperature = <115000>;
 };
 
 &tcphy0 {
@@ -542,7 +542,7 @@
 	rockchip,hw-tshut-mode = <1>;
 	/* tshut polarity 0:LOW 1:HIGH */
 	rockchip,hw-tshut-polarity = <1>;
-	rockchip,hw-tshut-temp = <110000>;
+	rockchip,hw-tshut-temp = <120000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
index 03a5f425a892..3d6e9e9215c0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
@@ -631,13 +631,13 @@
 };
 
 &soc_crit {
-	temperature = <100000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
 	rockchip,hw-tshut-mode = <1>; /* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
-	rockchip,hw-tshut-temp = <110000>;
+	rockchip,hw-tshut-temp = <120000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
index 8c50c23bd33c..5763c4931835 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rv1-android.dts
@@ -526,13 +526,13 @@
 };
 
 &soc_crit {
-	temperature = <105000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
 	rockchip,hw-tshut-mode = <1>; /* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
-	rockchip,hw-tshut-temp = <110000>;
+	rockchip,hw-tshut-temp = <120000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
index 3a4a345072aa..6a84991d1e34 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-vr-android.dts
@@ -716,13 +716,13 @@
 };
 
 &soc_crit {
-	temperature = <100000>; /* millicelsius */
+	temperature = <115000>; /* millicelsius */
 };
 
 &tsadc {
 	rockchip,hw-tshut-mode = <1>; /* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
-	rockchip,hw-tshut-temp = <110000>;
+	rockchip,hw-tshut-temp = <120000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 124a9114fb3d..d026527ee879 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -755,7 +755,7 @@
 					type = "passive";
 				};
 				soc_crit: soc-crit {
-					temperature = <95000>; /* millicelsius */
+					temperature = <115000>; /* millicelsius */
 					hysteresis = <2000>; /* millicelsius */
 					type = "critical";
 				};
@@ -807,7 +807,7 @@
 		pinctrl-1 = <&otp_out>;
 		pinctrl-2 = <&otp_gpio>;
 		#thermal-sensor-cells = <1>;
-		rockchip,hw-tshut-temp = <95000>;
+		rockchip,hw-tshut-temp = <120000>;
 		status = "disabled";
 	};
 
-- 
2.35.3

